--- pyempathygtk.defs	2008-02-10 17:56:26.000000000 +0100
+++ pyempathygtk/pyempathygtk.defs	2008-02-10 17:57:11.000000000 +0100
@@ -147,13 +147,6 @@
   (gtype-id "EMPATHY_TYPE_THEME_MANAGER")
 )
 
-(define-object Spinner
-  (in-module "Ephy")
-  (parent "GtkWidget")
-  (c-name "EphySpinner")
-  (gtype-id "EPHY_TYPE_SPINNER")
-)
-
 ;; Enumerations and flags ...
 
 (define-enum ChatViewBlock
@@ -2112,40 +2105,3 @@
   (varargs #t)
 )
 
-
-
-;; From ephy-spinner.h
-
-(define-function ephy_spinner_get_type
-  (c-name "ephy_spinner_get_type")
-  (return-type "GType")
-)
-
-(define-function ephy_spinner_new
-  (c-name "ephy_spinner_new")
-  (is-constructor-of "EphySpinner")
-  (return-type "GtkWidget*")
-)
-
-(define-method start
-  (of-object "EphySpinner")
-  (c-name "ephy_spinner_start")
-  (return-type "none")
-)
-
-(define-method stop
-  (of-object "EphySpinner")
-  (c-name "ephy_spinner_stop")
-  (return-type "none")
-)
-
-(define-method set_size
-  (of-object "EphySpinner")
-  (c-name "ephy_spinner_set_size")
-  (return-type "none")
-  (parameters
-    '("GtkIconSize" "size")
-  )
-)
-
-
