--- /home/zdra/Desktop/pyempathy.defs	2008-04-02 11:08:02.000000000 +0200
+++ pyempathy/pyempathy.defs	2008-04-02 11:28:00.000000000 +0200
@@ -44,6 +44,7 @@
 (define-object ContactManager
   (in-module "Empathy")
   (parent "GObject")
+  (implements "EmpathyContactList")
   (c-name "EmpathyContactManager")
   (gtype-id "EMPATHY_TYPE_CONTACT_MANAGER")
 )
@@ -121,6 +122,7 @@
 (define-object TpContactList
   (in-module "Empathy")
   (parent "GObject")
+  (implements "EmpathyContactList")
   (c-name "EmpathyTpContactList")
   (gtype-id "EMPATHY_TYPE_TP_CONTACT_LIST")
 )
